frontend/src/views/CellCards/components/CellCard/components/common/HelpTooltip/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Tooltip } from "czifui";
```

Comment:
This tooltip will be used to display extra information for the various data (marker genes, ontology view, enriched genes, etc)